xbmc/interfaces/json-rpc/PlayerOperations.cpp

```diff
@@ -1555,6 +1555,71 @@ JSONRPC_STATUS CPlayerOperations::GetPropertyValue(PlayerType player, const std:
         break;
     }
   }
+  else if (property == "currentvideostream")
+  {
+    switch (player)
+    {
+    case Video:
+      if (g_application.m_pPlayer->HasPlayer())
+      {
+        result = CVariant(CVariant::VariantTypeObject);
+        int index = g_application.m_pPlayer->GetVideoStream();
+        if (index >= 0)
```

Comment:
What happens with the result if `index` is smaller than `0`? Shouldn't it result in a `null` result as well?